data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/threshold/AbsoluteFlowThresholdTest.java

```diff
@@ -22,71 +22,125 @@
 
 /**
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ * @author Baptiste Seguinot {@literal <baptiste.seguino at rte-france.com>}
  */
 public class AbsoluteFlowThresholdTest {
 
+    private static final double DOUBLE_TOL = 0.5;
+
     private AbsoluteFlowThreshold absoluteFlowThresholdAmps;
     private AbsoluteFlowThreshold absoluteFlowThresholdMW;
     private Cnec cnec1;
     private Cnec cnec2;
+    private Cnec cnec3;
     private Network networkWithoutLf;
     private Network networkWithtLf;
```

Comment:
typo here, I think it should be networkWithLf

Reply:
you're right, corrected